lib/util/error.js

```diff
@@ -816,6 +828,11 @@ shaka.util.Error.Code = {
    */
   'OBJECT_DESTROYED': 7003,
 
+  /**
+   * The player has not been initialized so the operation is not supported.
```

Comment:
Since there isn't an "initialize" step or method in the Player that applications would see, I think it might be best to use different terms for this.  We would probably say something like "content has not been loaded in the Player".  The error constant's name should be changed, as well.  Thanks!